app/components/layout/sidebar/multi_level_menu_component.html.erb

```diff
@@ -0,0 +1,71 @@
+<div data-controller="collapsible">
+  <button
+    type="button"
+    class="
+      flex
+      w-full
+      items-center
+      mt-1
+      p-3
+      transition-colors
+      ease-in-out
+      delay-75
+      hover:bg-slate-200
+      dark:text-slate-200
+      dark:hover:bg-slate-700
+    "
+    aria-controls="<%= "multi-level-menu_#{title.parameterize.underscore}"%>"
```

Comment:
**nitpick:** Wouldn't it make more sense to create a variable so it can be used in both places? 
`<% control_id = "multi-level-menu_#{title.parameterize.underscore}" %>`

Reply:
That made a little too much sense :) Updated in ee51805f2f610a8568af3421cc99e1992a6fbf41